packages/truffle-migrate/index.js

```diff
@@ -23,10 +23,14 @@ const Migrate = {
   },
 
   assemble: function(options, callback) {
+    var config = Config.detect(options);
```

Comment:
@roderik , when building on my own I get `ReferenceError: Config is not defined`

Reply:
Missing `const Config = require("truffle-config");` (note the const)